.github/workflows/workflows.yaml

```diff
@@ -336,6 +505,15 @@ jobs:
         run: |
           echo "VERSION=${{ steps.get_version.outputs.version }}" >> $GITHUB_ENV
 
+      - name: Get git tag version Debian arm64 # might be redundant
```

Comment:
I am still not understanding these comments - could we clarify these before we merge?
My suggestion is to simply remove the comments, or specify a TODO.

Reply:
Resolved. As @olof-nord found out, battila7/get-version-action@v2 produces the tag in this form "v1.0.9" and after removing the "v" it's usable.